news/history-all.rst

```diff
@@ -0,0 +1,50 @@
+**Added:** None
```

Comment:
I'll go ahead and do that  I'll merge in a couple of minutes after playing around with it.